app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -69,37 +62,31 @@ If your application is successful, the training provider may be able to help you
 
 Most schools in England need you to have [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach. 
 
-Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 
+Without QTS, you will not be eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
 
-Without QTS, you will not eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
+Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 
 
 ## If the course is fee-funded or salaried
 
 ### Fee-funded teacher training
 
-Most teacher training courses are fee-funded – this means you have to pay tuition fees and will not earn a salary while you train. 
+Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 
 
 There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship.
 
 You can [find out about the different ways to fund your training](/funding-and-support).
 
 <div class="inset">
 
-<p> If you’re a non-UK citizen without indefinite leave to remain in the UK, you will not usually be eligible for a bursary or scholarship, unless you train to teach languages or physics.</p>
+<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. Other non-UK citizens will not be eligible for student finance, but may be eligible for a bursary or scholarship if they train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>
```

Comment:
```suggestion
<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. Other non-UK citizens are not eligible for student finance, but you may be eligible for a bursary or scholarship if you train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>
```
This feels like a lot of information - worth splitting into two paras?

Also it feels odd that we talk about `you` and `they`?

Finally GDS guidance is to avoid anchor links (https://www.gov.uk/guidance/content-design/links) - whilst we might need to keep this for now, this is something we should look to phase out as we do the integration work